src/Sulu/Bundle/SecurityBundle/UserManager/UserManager.php

```diff
@@ -681,7 +681,7 @@ private function processEmail(UserInterface $user, $email, $contact = null)
             }
         } else {
             if (null !== $email) {
-                if ($email !== $user->getEmail() &&
+                if (strtolower($email) !== strtolower($user->getEmail()) &&
```

Comment:
fixes #5596

Reply:
Think a test for this fix would be great 👍

Reply:
Yes I havent found time to add any tests yet. Will add them when we agree on how to implement it 🙂

Reply:
Reverted this change because of a failing test case. Guess we need to fix this in another PR 🙂